lib/Position/WeatherParserUtils.js

```diff
@@ -0,0 +1,78 @@
+
+
+const wind_multiplier = 0.44704;
+const rain_multiplier = 0.254;
+const weatherRegEx = new RegExp(/([cSgtrpPlLs#]\d{3}|t-\d{2}|h\d{2}|b\d{5}|s\.\d{2}|s\d\.\d)/g);
```

Comment:
Problem I see with using regex is it doesn't ensure that all fields are together. For example "t000Testh02" both t/h will be parsed and there will be no comment in output ("Test" will be removed) I think it's not really desirable and Testh02 should be treated as comment.

Reply:
Adding another test case made one fail due to this regex, so good catch :).